src/quicktype-core/Renderer.ts

```diff
@@ -266,10 +290,20 @@ export abstract class Renderer {
         return assignNames(this.setUpNaming());
     }
 
+    protected initializeEmitContextForFilename(filename: string): void {
+        if (this._finishedEmitContexts.has(filename.toLowerCase().toString())) {
+            const existingEmitContext = this._finishedEmitContexts.get(filename.toLowerCase().toString());
```

Comment:
`.toLowerCase().toString()`?  Is the `toLowerCase()` to deal with case-insensitive file systems?  Will the code work with case-sensitive file systems in all cases, too?

Reply:
Yes, it will.